compose.go

```diff
@@ -76,6 +76,56 @@ func (p *Plugin) CobraAddCommands(rootCmd *cobra.Command) error {
 	composeCmd.Flags().BoolVar(&conflictsVerbosity, "conflicts-verbosity", false, "Log files conflicts")
 	composeCmd.Flags().BoolVar(&clean, "clean", false, "Remove .compose dir on start")
 	composeCmd.Flags().BoolVar(&interactive, "interactive", true, "Interactive mode allows to submit user credentials during action")
+
+	composeDependency := &compose.Dependency{}
+	var createNew bool
+	var addCmd = &cobra.Command{
+		Use:   "compose:add",
+		Short: "Add new entry to plasma-compose",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.AddPackage(createNew, composeDependency, p.wd)
+		},
+	}
+
+	var updateCmd = &cobra.Command{
+		Use:   "compose:update",
+		Short: "Update plasma-compose entry",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// Don't show usage help on a runtime error.
+			cmd.SilenceUsage = true
+
+			return compose.UpdatePackages(p.wd)
```

Comment:
```
➜ ./plasmactl compose:update --help
Update plasma-compose entry                                                        

Usage:
  plasmactl compose:update [flags]

Flags:
  -h, --help   help for compose:update

Global Flags:
      --keyring-passphrase string   Passphrase for keyring encryption/decryption
  -q, --quiet                       disable stdOut
  -v, --verbose count               log verbosity level, use -vvv DEBUG, -vv WARN, -v INFO
```

This command should have options to edit a package entirely with CLI, without passing through form:
- packages
- source-type
- url
- tag
- ref